src/nrc/datamodel/models.py

```diff
@@ -87,7 +88,7 @@ class Meta:
         verbose_name_plural = _("abonnementen")
 
     def __str__(self) -> str:
-        return f"{self.uuid}"
+        return f"{self.callback_url}"
```

Comment:
I can't find this change in the scope of the user story

Reply:
I thought this comment, maybe I am wrong 
https://github.com/open-zaak/open-notificaties/issues/180#issuecomment-2548130622